elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HibernateEntityManagerStore.java

```diff
@@ -10,40 +10,22 @@
 
 import org.hibernate.ScrollMode;
 import org.hibernate.Session;
-import org.hibernate.jpa.HibernateEntityManager;
 
 import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
 import javax.persistence.metamodel.EntityType;
 
 /**
  * Hibernate5 store supporting the EntityManager.
  */
 public class HibernateEntityManagerStore extends AbstractHibernateStore {
-    protected final EntityManager entityManager;
+    protected final EntityManagerFactory entityManagerFactory;
 
-    @Deprecated
-    public HibernateEntityManagerStore(HibernateEntityManager entityManager,
+    public HibernateEntityManagerStore(EntityManagerFactory entityManagerFactory,
                                        boolean isScrollEnabled,
                                        ScrollMode scrollMode) {
         super(null, isScrollEnabled, scrollMode);
-        this.entityManager = entityManager;
-    }
-
-    public HibernateEntityManagerStore(EntityManager entityManager,
```

Comment:
Here too?

    @Deprecated
    public HibernateEntityManagerStore(EntityManager entityManager,
                                       boolean isScrollEnabled,
                                       ScrollMode scrollMode) {
        this(entityManager.getEntityManagerFactory(), isScrollEnabled, scrollMode);
    }
```